Sources/PushNotifications.swift

```diff
@@ -37,6 +37,8 @@ import Foundation
         } catch {
             print("Unexpected error: \(error).")
         }
+
+        self.sendMetadata()
```

Comment:
This looks like it's always sending metadata, but that will only happen if necessary.

Maybe the method should be `syncMetadata` instead? As it will sync if outdated but not send anything otherwise.

Reply:
I like your proposal 👍 
Nice one!